graphql-dgs-platform/build.gradle.kts

```diff
@@ -51,6 +51,9 @@ dependencies {
         api("com.graphql-java:graphql-java-extended-scalars") {
             version { require(Versions.GRAPHQL_JAVA_EXTENDED_SCALARS) }
         }
```

Comment:
I'm wondering of this should be added by default, or just be available as an opt-in by the user.